awx/main/models/ha.py

```diff
@@ -160,7 +160,7 @@ def __str__(self):
         default=100,
         editable=False,
     )
-    capacity_adjustment = models.DecimalField(default=Decimal(1.0), max_digits=3, decimal_places=2, validators=[MinValueValidator(0)])
+    capacity_adjustment = models.DecimalField(default=Decimal(1.0), max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal(0.0))])
```

Comment:
Does this not require a new migration file?

Reply:
```
bash-5.1$ awx-manage makemigrations
No changes detected
```
Does not appear so

Reply:
Don't think it actually changes the datatype, but rather validator function in Django

Reply:
validators had some weird place in both the fields and the model Meta. Weird but possible.